scripts/.env.example

```
@@ -0,0 +1,11 @@
+# Benchmark Configuration
+BASE_URL=http://localhost:3005
+TYPE=eoa
+FROM=0x...
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The FROM key should go before the TYPE key

(UnorderedKey)